### PR TITLE
Run E2E tests locally instead of in Docker

### LINKED ISSUE #681: Can't run E2E tests locally
I'm following the instructions in the [README](https://github.com/CDCgov/prime-simplereport/blob/main/README.md#e2e-tests) to run the e2e tests locally.

Looks like the app is failing to boot, it loops around this line: https://github.com/CDCgov/prime-simplereport/blob/2d3562827eaa7a52f5f60f511dec39d01aa1ba80/frontend/e2e/nightwatch-entrypoint.bash#L13

### PR DESCRIPTION
**Pull request by ClydeDroid**
## Related Issue or Background Info

Resolves #681 

## Changes Proposed

- Instead of running E2E tests in Docker, run them locally directly using Nightwatch

## Additional Information

- The default `yarn e2e` command uses Firefox, so it assumes you have it installed
- You can run them _not_ headless (ie you can watch them run in browser) by commenting out `--headless` in `nightwatch.conf.js`
- This _should_ work with chrome, using `yarn e2e:chrome` (I haven't gotten a chance to test)
- Also updates the CI job to install Firefox

### CODE REVIEW

**Comment by ClydeDroid** [comment]
Hmmm, looks like there's an issue w/ time zones for the birth dates causing a difference between local and CI envs - sorting that out now

**Review by nullflux — APPROVED** [review]
This works locally for me with Firefox!

**Review by timothybest-usds — APPROVED** [review]
I pushed a commit to get this working in chrome. Long term we will need to takle https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/432 and get cross browser date & date time inputs working

File: frontend/e2e/tests/full-suite.js [hunk 1]
```diff
@@ -1,12 +1,22 @@
 let patientName, birthDate;
 
+
+const getDobFormat = (browser) => {
+  /* NOTE: DOB format is currently browser specific
+       - Firefox does not support <input type='date' /> and takes the format accepted by the app
+       - Chrome does support the date type and expects MM/DD/YYYY
+  */
+  return browser.capabilities.browserName === 'chrome' ? 'MM/DD/YYYY' : 'YYYY-MM-DD';
```
Comment (ClydeDroid) [inline frontend/e2e/tests/full-suite.js:9]: :facepalm: well isn't that fun! Thanks for the fix :+1:

**Pull request merged**